### DISCUSSION TITLE
Is there some reason YouTube video "start at" seconds cannot be passed with content/video/my-post.md ?

### DISCUSSION THREAD

**Discussion by darrenaddy** [Lednerb/bilberry-hugo-theme#367] [Q&A]
I'm able to see the video when I create a post using
`hugo new content/video/my-post.md`

However, if I make the front matter, for example:
`youtube: "OweuMTg1nbQ?t=30"`
It does not jump one to the 30 second mark of the video to start. Is this by design or should it be a feature request, perhaps??

ASIDE: One gotcha that I did not see explained (for the video content type) is that you need to DELETE all of the frontmatter for types that you are not using. It will not simply ignore empty values because it uses a series of IF statements (rather than using ELSE statements after each IF). It will display the YouTube video, but will also insert a bunch of space with a PLAY button **under** the YouTube video until you delete that unused front matter.

Thanks everyone for this great theme!

**Comment by igor-baiborodine** [comment]
@darrenaddy, It will not work since the source URL for YouTube that is used in the video-embed.html partial contains the `rel` parameter:
https://www.youtube-nocookie.com/embed/{{ .id }}?rel=0. 

When you append a new parameter to the video ID, it will be ignored: https://www.youtube-nocookie.com/embed/{{ .id }}?start=70?rel=0.

Therefore, the URL below works:
https://www.youtube-nocookie.com/embed/OweuMTg1nbQ?start=70

But that one doesn't:
https://www.youtube-nocookie.com/embed/OweuMTg1nbQ?start=70?rel=0

>ASIDE: One gotcha that I did not see explained (for the video content type) is that you need to DELETE all of the frontmatter for types that you are not using. It will not simply ignore empty values because it uses a series of IF statements (rather than using ELSE statements after each IF). It will display the YouTube video, but will also insert a bunch of space with a PLAY button under the YouTube video until you delete that unused front matter.

For this gotcha, I will update the documentation.
